### ISSUE TITLE
Number got banned after 5 messages

### ISSUE THREAD

**Issue by felri** [Kalebu/alright#136]
Just a heads up I used the example.py in a loop and got banned in 5 messages, not an issue, just a warning.

**Comment by juandres666** [comment]
> Just a heads up I used the example.py in a loop and got banned in 5 messages, not an issue, just a warning.

Hello, how much time do I need to wait if I don't want to get banned? thanks

**Comment by theCJMan** [comment]
Hi there
I have added a random sleep between random messages..
2 variables, random number of messages to send, and after that wait for a random time, AND dont send more than XXX per 24 hours... Not sure what te exact numbers are from whatsapp

Naturally, also, if your contacts mark you as a spammer, then you are stuffed in any case...

I sent yesterday over 600 messages successfully, BUT!!!! this morning the number was blocked as spammer...
Now I do not know if this is because someone marked me as a spammer...
AND!!!
I waited 10 sec between every send as I sent large videos to allow the videos to upload..

**Comment by iamtalhaasghar** [comment]
> Hi there I have added a random sleep between random messages.. 2 variables, random number of messages to send, and after that wait for a random time, AND dont send more than XXX per 24 hours... Not sure what te exact numbers are from whatsapp
> 
> Naturally, also, if your contacts mark you as a spammer, then you are stuffed in any case...
> 
> I sent yesterday over 600 messages successfully, BUT!!!! this morning the number was blocked as spammer... Now I do not know if this is because someone marked me as a spammer... AND!!! I waited 10 sec between every send as I sent large videos to allow the videos to upload..

were you sending msgs to existing contacts or new ones? I got blocked after 20 msgs to new numbers and account was also new. I had i think 10 or 15 secs wait before sending another msg.

**Comment by iamtalhaasghar** [comment]
related https://github.com/iamtalhaasghar/alright/commit/b3237a96c2574dfcb96a00179dbc9093db9a98d6#commitcomment-143075864

**Comment by theCJMan** [comment]
Remember
Any contact can block and report you as spam. 
It is therefore not just a function of how you send but also if the recipients mark you as spam.

⁣--------------------------------
Cornelius Mostert
+27 79 099 5412

Get BlueMail for Android ​

**Comment by iamtalhaasghar** [comment]
@theCJMan 
Yeah, correct but if you send msgs to too many new contacts compared to existing contacts that also raises a suspicion. So my original question was were you sending msgs to people you had already chatted with or they were new contacts?

**Comment by theCJMan** [comment]
8k mixed contacts mostly new, with all the randomness built in..

⁣--------------------------------
Cornelius Mostert
+27 79 099 5412

Get BlueMail for Android ​
